andela/trime
# Bug/create project

#### What does this PR do?

Fixes a bug in the org page with the `Create Project` button. A new project was not being created on clicking `Submit`

#### Description of Task to be completed?

Enable a user to create a new project from an org's page.

#### How should this be manually tested?

After logging in and creating an org click on the org title to go the org's page. An `Add Project` button will be on the upper right corner, clicking on it and filling in the relevant details then `Submit` should create a new project in the organisation.

#### What are the relevant pivotal tracker stories?

[Bug #126112963] (https://www.pivotaltracker.com/n/projects/1454736/stories/126112963)

@andela-ladenusi
